src/main/kotlin/io/github/mojira/arisa/modules/commands/FixedCommand.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.mojira.arisa.modules.commands
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import io.github.mojira.arisa.domain.Issue
+import io.github.mojira.arisa.modules.ModuleError
+import io.github.mojira.arisa.modules.ModuleResponse
+import io.github.mojira.arisa.modules.OperationNotNeededModuleResponse
+import io.github.mojira.arisa.modules.assertTrue
+
+class FixedCommand : Command {
+    override fun invoke(issue: Issue, vararg arguments: String): Either<ModuleError, ModuleResponse> = Either.fx {
+        assertTrue(arguments.size > 1).bind()
+        val version = arguments[1]
```

Comment:
```suggestion
        val version = arguments.asList().subList(1, arguments.size).joinToString(" ")
```
Probably same as for AddVersionCommand.kt to handle version names with spaces.